### ISSUE TITLE
Oven Smoke Perma Depletes Stamina, Can't Catch Breath

### ISSUE THREAD

**Issue by SibilantScream** [CleverRaven/Cataclysm-DDA#48725]
![image](https://user-images.githubusercontent.com/61993103/116790202-c1aed980-aabb-11eb-90b9-d1d3d55c5495.png)
I crafted clean water & the smoke just got half of my HP & permanently zeroed my breath. I tried waiting to catch breath so many times the sun went down.

Platform     : Windows_x64
Graphics     : Tiles
Build number : #11581
Build version: 0.E-10395-g4525b4f
Major version: 0.E

**Comment by danielkwinsor** [comment]
I have this issue with any type of smoke.

**Comment by actual-nh** [comment]
I'm assuming the character(s) don't have asthma, which would make that rather more realistic?

**Comment by SibilantScream** [comment]
No asthma.

**Comment by RoyBerube** [comment]
Pus filled torso might have something to do with it.

**Comment by actual-nh** [comment]
> Pus filled torso might have something to do with it.

@danielkwinsor - screenshot when you encounter this again?

**Comment by SibilantScream** [comment]
did a fresh new run w\ same start. Same thing occured, even before bad infection stat:
![image](https://user-images.githubusercontent.com/61993103/116960615-9c7fbe00-aca9-11eb-9a20-93d9748ec20c.png)

**Comment by actual-nh** [comment]
@Venera3: Is infection supposed to have this effect at that stage?

**Comment by Venera3** [comment]
Your stamina still regens up till Pus-filled (at Infected walking is still stamina-positive, at Bady Infected it becomes stamina-neutral and regen zeroes out at Pus-Filled). I'll take a look at the smoke effect to see if there's an unintended interaction.

**Comment by Venera3** [comment]
Yeah, it's an unhappy interaction. Coughing is hardcoded, with a non-harmful variant that only costs moves and a harmful one that reduces your stamina by 5% of your max (for most characters 50/10,000) and damages you by the difference if you'd go negative -- `character.cpp/9038`, in case somebody more fluent wants to double check. The smoke effect causes harmful coughing every 10th second effectively, depleting your stamina pretty quickly and the infection prevents you from regenerating at high intensities.

**Comment by actual-nh** [comment]
> Yeah, it's an unhappy interaction. Coughing is hardcoded, with a non-harmful variant that only costs moves and a harmful one that reduces your stamina by 5% of your max (for most characters 50/10,000) and damages you by the difference if you'd go negative -- `character.cpp/9038`, in case somebody more fluent wants to double check.

Eventually, it may be a good idea to replace that with intermittent "winded" plus the sound production #48671 should enable.

**Comment by SibilantScream** [comment]
Not sure about exertion advancing infection since you need to find antibiotics fast but I agree on over-exertion leading to exhaustion & starvation\dehyderation penalties to simulate losing energy.
Losing HP directly to smoke is just not inelegant & unrealistic, having even a hidden soffocation value that's shared between being submerged & smoked among other situations is more intuitive.

**Comment by danielkwinsor** [comment]
<img width="972" alt="Screenshot 2021-05-08 182633" src="https://user-images.githubusercontent.com/2822992/117557747-15c13d00-b02b-11eb-807a-77f4979e4bd9.png">
